lib/githubModel.js

```
@@ -0,0 +1,70 @@
+    // Validate that the API response has the expected structure
+    if (!response.data.choices || !response.data.choices[0]?.message?.content) {
+      throw new Error('Invalid response format from GitHub model API');
+    }
+
+    const rawContent = response.data.choices[0].message.content.trim();
+
+    // Clean the response if it's wrapped in markdown code blocks
+    let cleanedContent = rawContent;
+    if (rawContent.startsWith('```') && rawContent.endsWith('```')) {
+      const match = rawContent.match(MARKDOWN_CODE_BLOCK_REGEX);
+      if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)